PROTO_tests/tests/io_util_test.py

```diff
@@ -0,0 +1,47 @@
+import os
+import shutil
+from pathlib import Path
+
+import pytest
+
+from arkouda import io_util
+
+
+class TestIOUtil:
+    @classmethod
+    def setup_class(cls):
+        cls.io_test_dir = "{}/io_util_test/".format(os.getcwd())
```

Comment:
I think we should update this and the other spots in this file to use `f` strings instead
```python
cls.io_test_dir = f"{os.getcwd()}/io_util_test/"
```

Reply:
Updated